src/main/java/frc/robot/RobotContainer.java

```diff
@@ -53,6 +65,10 @@ public class RobotContainer {
 
   // Controller
   private final CommandXboxController controller = new CommandXboxController(0);
+  // private final CommandXboxController controller = new CommandXboxController(0);
+  private final Joystick Joystick1 = new Joystick(0);
+  private final Joystick Joystick2 = new Joystick(1);
+  private final Joystick CoJoystick = new Joystick(2);
```

Comment:
```suggestion
  private final Joystick joystick1 = new Joystick(0);
  private final Joystick joystick2 = new Joystick(1);
  private final Joystick coJoystick = new Joystick(2);
```

⛏️ Usually good form to have these be lowercase.